docs/docs/configuration/mapper.mdx

```diff
@@ -68,6 +68,37 @@ public partial class CarMapper
 }
 ```
 
+#### IgnoreAttribute
+
+To ignore a property or field at definition, the `MapperIgnoreAttribute` can be used.
+
+```csharp
+public partial class CarMapper
+{
+    public partial CarDto ToDto(Car car);
+}
+
+public class Car
+{
+    // highlight-start
+    [MapperIgnore]
+    // highlight-end
+    public int Id { get; set; }
+
+    public string ModelName { get; set; }
+}
+
+public class CarDto
+{
+    // highlight-start
+    [MapperIgnore]
+    // highlight-end
+    public int MakeId { get; set; }
```

Comment:
I think the example would be easier to understand if this property is removed. Otherwise one could be thinking the `MapperIgnoreAttribute` is required on both sides, the source and the target class.